### PR TITLE
Forensics Trainee its they own job and Buff the janitor belt

### PR DESCRIPTION
**Pull request by Guidesu**
Due to this person be EXTREMELY dumb at coding, I don't know how to remove my other commits.

Ignore the Sergal lore thing, I was messing around trying to learn a few things.

Jani belt can hold galoshes and trash bags.
I don't see the reason why.
You can even put a trash bag in a webbing 

I don't see the reason why Forensics trainee was an alt title.
So they get the own security icon and the same access as the Forensics Technician
They obey ONLY the Forensics Technician and COS or people who outrank they

🆑 Jani belts now hold trash bags and galoshes! Rejoice!
ADD: Forensics trainee is their own job now, they get their own locker and access to forensics.
REMOVE: Remove Forensics trainee alt title to Security cadet. Dead caput!
ADD: ADD an Alt title to Security cadet. Cadet...that it. Cadet
REMOVE: Remove the useless locker to add the locker to the cadet in the lab area
image

### CODE REVIEW

**Comment by Guidesu** [comment]
i think i forgot the ,

**Comment by Guidesu** [comment]
proof its works
https://cdn.discordapp.com/attachments/762930148101849110/783075398048808970/unknown.png

**Comment by Guidesu** [comment]
I need someone to teach me how to use GitHub badly.
I wanted to make another pull request to the forensics trainee but I merged the two in one.
Please if you tell me what i can improve i would be REALLY happy
This is my first BIG project using DM.
I still learning how to code

**Comment by Guidesu** [comment]
https://cdn.discordapp.com/attachments/783039077565202433/783069932191219722/unknown.png
Oh yes i suffered ALOT to make the damn sechud icon a thing

**Comment by Guidesu** [comment]
Oh...wow just one failed check..
I feel proud

File: maps/torch/job/security_jobs.dm [hunk 1]
```diff
@@ -164,6 +164,64 @@
...
+	alt_titles = list(
+		"Detective Apprentice"
+		)
+	outfit_type = /decl/hierarchy/outfit/job/security/detective
+
+
+	outfit_type = /decl/hierarchy/outfit/job/torch/crew/security/forensic_tech
+	allowed_branches = list(
+		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/security/forensic_tech/fleet,
+		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/security/forensic_tech/marine
```
Comment (BurpleBineapple) [inline maps/torch/job/security_jobs.dm:190]: This should be /datum/mil_branch/marine, duplicated

Comment (BurpleBineapple) [inline maps/torch/job/security_jobs.dm]:
```suggestion
		/datum/mil_branch/marine = /decl/hierarchy/outfit/job/torch/crew/security/forensic_tech/marine
```

File: code/modules/boh/culture_descriptor/faction/factions_sergal.dm [hunk 2]
```diff
@@ -16,6 +16,7 @@
 	Since the year of the Rain-Calendar 20, it was unified as the name of the Kingdom and Rain Silves' clan, formed from a conglomerate of unified tribes under one flag.\
 	The 'Shigu Kingdom' collapsed during the late stage of world war. And after the end of the war, it reconstructed as the 'Shigu Dukedom', developed as a large city of Northern Tal."
 
+
```
Comment (BurpleBineapple) [inline code/modules/boh/culture_descriptor/faction/factions_sergal.dm]:
```suggestion

```

Comment (BurpleBineapple) [reply code/modules/boh/culture_descriptor/faction/factions_sergal.dm]: Random spaces bad

Comment (Guidesu) [reply code/modules/boh/culture_descriptor/faction/factions_sergal.dm]: yup.
I accidentally messed with this part of the code

**Comment by Guidesu** [comment]
Also, i need confirmation from you.
Several peoples wish to the DA be SFP or Contractor.
Would you accept?
Can i start coding?
A contracted FT would bring along a contracted FTA
Still, I think the idea of an SFP agent being an apprentice don't make much sense


Suggestion: Allow Detective Apprentice to be SFP/Contractor. Since FT can already be NTEF, SMC, Contractor, SFP, and Civilian, why not let their less experienced half be one as well.

**Comment by Guidesu** [comment]
image
Table

**Comment by BurpleBineapple** [comment]
I would say no to making trainee roles available to anything outside of uniformed branches. Only NTEF/SMC should be in trainee roles. No civilians, no contractors.

**Comment by Guidesu** [comment]
Well if you want to discuss anything about my PR
DM on discord i got muted for about 12 hours.
Nice.
I am not even bad its just sad.

Anyway , i will modify a few things.
I don't think it have bugs.
Have a good day purple

File: maps/torch/job/security_jobs.dm [hunk 3]
```diff
@@ -164,6 +164,64 @@
...
+		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/security/forensic_tech/marine
+
+	)
+
+
+
+	min_skill = list(   SKILL_EVA         = SKILL_BASIC,
+	                    SKILL_COMBAT      = SKILL_BASIC,
+						SKILL_WEAPONS     = SKILL_BASIC,
+						SKILL_FORENSICS   = SKILL_AVERAGE,)
```
Comment (Firefox13) [inline maps/torch/job/security_jobs.dm]: изображение
choose one of these:
изображение
SKILL_AVERAGE will result on lack of the skill. Gee, did you even test it?

Comment (Guidesu) [reply maps/torch/job/security_jobs.dm]: Yes, i tested...
How i saaaaaaid i am new.
And i think it worked in game.
And i didn't found the files to define...
Ack sorry

Comment (Guidesu) [reply maps/torch/job/security_jobs.dm]: Also FireFox.
For some reason.
The average skill WORKED on the game.
I SWEAR

Comment (Guidesu) [reply maps/torch/job/security_jobs.dm]: Now its fixed

File: maps/torch/job/security_jobs.dm [hunk 4]
```diff
@@ -131,13 +131,13 @@
 	total_positions = 2
 	spawn_positions = 2
 	supervisors = "the entirety of Security"
+	alt_titles = list(
+		"Cadet"
+		)
```
Comment (io-util) [inline maps/torch/job/security_jobs.dm]: Don't see why there's a need for this alt title.

File: maps/torch/job/security_jobs.dm [hunk 5]
```diff
@@ -164,6 +164,65 @@
...
+		/datum/mil_rank/fleet/e3,
+		/datum/mil_rank/marine_corps/e2,
+		/datum/mil_rank/marine_corps/e3
+	)
+
+	access = list(access_security, access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_eva)
+	minimal_access = list(access_security, access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_eva)
+
+/datum/job/forencadet/get_description_blurb()
+	return "You are a new cadet onboard [GLOB.using_map.full_name]. Make sure to Obey the Forensic Technician and the chief of security"
```
Comment (io-util) [inline maps/torch/job/security_jobs.dm]:
```suggestion
	return "You are a new Forensics Technician in training aboard the [GLOB.using_map.full_name]. Make sure to obey the Forensic Technician and the Chief of Security."
```

**Review by io-util — CHANGES_REQUESTED** [review]
Make sure to squash all of the commits into one.

File: maps/torch/job/security_jobs.dm (hunk 4 above)
Comment (Guidesu) [reply maps/torch/job/security_jobs.dm]: right

Comment (Guidesu) [reply maps/torch/job/security_jobs.dm]: removed

**Comment by Guidesu** [comment]
Loorey what is merge branch?
I think i clicked in one thing

**Comment by Guidesu** [comment]
also how i squash everything?
I have ZERO idea how

**Comment by Guidesu** [comment]
I really cant see how the fuck i squash all commits in one...

**Comment by Guidesu** [comment]
I almost giving up because i dont really understand github

**Comment by Guidesu** [comment]
If i had any idea what i am doing

**Comment by Guidesu** [comment]
You see....
I will try to learn more and how to use github...
I would be glad if any of you taught me how to use github.
I will close this  project because god knows why i am so stupid

**Comment by Guidesu** [comment]
Lamasmaster told me you can squash all commits ...
I hope you can because i dont

**Pull request closed without merging**